server/server.js

```diff
@@ -1619,13 +1619,13 @@ async function afterLogin(socket, user) {
     socket.join(user.id);
 
     let monitorList = await server.sendMonitorList(socket);
-    sendInfo(socket);
-    server.sendMaintenanceList(socket);
-    sendNotificationList(socket);
-    sendProxyList(socket);
-    sendDockerHostList(socket);
-    sendAPIKeyList(socket);
-    sendRemoteBrowserList(socket);
+    await sendInfo(socket);
+    await server.sendMaintenanceList(socket);
+    await sendNotificationList(socket);
+    await sendProxyList(socket);
+    await sendDockerHostList(socket);
+    await sendAPIKeyList(socket);
+    await sendRemoteBrowserList(socket);
```

Comment:
In my opinion, using `await` or not is depending on the design. The need of blocking and non-blocking.

Here, most of them are actually intended not to use `await`, because they are independent. And for MariaDB, they can query at the same time without doing one by one.

Reply:
Agreed. But also, the final user experience will still depend on many factors. For example, if we are IO-bound on the server-side, there is no benefit to blasting off the responses all at once, and awaiting makes the application behavior more predictable. Having to `sleep(500)` seems crude in comparison.

Reply:
Maybe something like `Promise.all()` would be a good compromise?

Reply:
I think in this case `Promise.allSettled()` might be better, but yes (am going to propose another PR incl. remmoving the `sleep`)

I was afraid of running into weird unreproducible race conditions, but in this case this is likely ok ^^